hack/make/test-unit

```diff
@@ -48,8 +48,8 @@ bundle_test_unit() {
 			| grep -v github.com/docker/docker/man \
 			| grep -v github.com/docker/docker/integration-cli)
 	fi
-
 	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS $pkg_list
+	go test -cover -ldflags "$LDFLAGS" "${BUILDFLAGS[@]}" $TESTFLAGS github.com/docker/docker/pkg/loopback -test.root
```

Comment:
@AkihiroSuda: I suggested adding this flag because I feel that `go test <packagename>` without extra arguments should work without expecting a certain environment or level of privilege. It would be unexpected for someone who downloads this package to run `go test` and have the tests fail because `go test` was not run as root, or, worse, to have the test mess with loop devices on their system.

The "requires root" flag is the way containerd's tests deal with this: https://github.com/containerd/containerd/blob/master/testutil/helpers.go

I realize there are existing tests that assume root privileges and interact with system resources, but rather than adding to the problem I think we should migrate those tests to use this new flag.